src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+@import "../utilis/variables";
+
+.cards-container {
+  display: grid;
+  justify-content: center;
+  gap: 46px 48px;
+  grid-template-columns: repeat(var(--columns),200px);
+  --columns: 1;
+  padding: 50px 0 72px;
+}
+
+.card {
+  max-width: 200px;
+  width: 100%;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  transition: 0.3s;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 36px 3px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: $thinner-font-weight;
+    line-height: $bigger-line-height;
+    color: $text-dark;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-size: $default-font-size;
+    line-height: $default-line-height;
+    color: $text-gray;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    font-size: $default-font-size;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-amount {
+      font-size: 16px;
+      line-height: $bigger-line-height;
+      font-weight: $bolder-font-weight;
+      color: $text-dark;
+    }
+
+    &-caption {
+      font-size: 12px;
+      line-height: $bigger-line-height;
+      color: $text-gray;
+    }
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 14px;
+    line-height: $default-line-height;
+    font-weight: $bolder-font-weight;
+    color: $color-white;
+    background-color: $color-blue;
+    text-decoration: none;
```

Comment:
you can also reset that, it's rarely the case to have `text-decoration` anything other than `none` 😄